pyproject.toml

```
@@ -60,6 +60,7 @@ path = ".venv"
 test        = "pytest -n auto --cov src --cov-report=xml tests/unit"
 coverage    = "pytest -n auto --cov src tests/unit --cov-report=html"
 integration = "pytest -n 10 --cov src tests/integration"
+performance = "pytest -n 10 --cov src tests/performance"
```

```suggestion
performance = "pytest --cov src tests/performance"
```
No need for multiple workers here :)